README.md

```diff
@@ -111,6 +111,17 @@ You can check for possible unused code with `vulture` by running:
 It tends to produce a lot of false positives, but it can provide a good place
 to start.
 
+To identify simple issues before submission to code review using `pre-commit`,
+install it and set up the git hook scripts by running:
```

Comment:
Can we change the wording to

> There are some `pre-commit` hooks that can fix basic formatting issues for you to make the review process go smoother. You can install them by running:

I would also put the section about `pre-commit` before the section about vulture, since vulture is a kindof niche thing.